### PR TITLE
Adding RSVP Keeper to the Apps/Websites section

### PR DESCRIPTION
**Pull request by johndatserakis**
Adding RSVP Keeper to the Apps/Websites section

RSVP Keeper is a full-featured online reservation app built with Vue and Go.

https://www.rsvpkeeper.com/

### CODE REVIEW

**Comment by johndatserakis** [comment]
Hey @Scrum, I just merge in the latest from master. Everything's merged in and ready to go.

Thank you.

File: README.md [hunk 1]
```diff
@@ -976,9 +976,10 @@
   - [Translator-vuejs](https://manuel-suarez-abascal.github.io/translator-vuejs/) - Translation App built with Vuejs, Yandex API & ResponsiveVoice.js API.
   - [Big Timer](https://www.bigtimer.net/) - Fullscreen countdown timer for workshops, meetings and presentations. Big Timer helps workshop facilitators, meeting chairs, design sprinters, presenters and aspiring game show hosts stick to their program.
   - [Wirenook](https://wirenook.com/) - Free online app for building responsive website wireframes. Hi- and Lo- Fidelity, project sharing and svg download.
+  - [RSVP Keeper](https://www.rsvpkeeper.com/) - Online reservations made easy. Get your event up and running in no time. Made with Vue and Go.
```
Comment (Scrum) [inline README.md]: @johndatserakis Needs to be at the end of the list.

File: README.md [hunk 2]
```diff
@@ -976,9 +976,10 @@
...
   - [Kvalitetskontroll](https://www.kvalitetskontroll.no/) - Norwegian management system tailored for the construction industry.
   - [Poolside FM](https://poolside.fm/) - A retro-style music player
-  - [Inoreader.com](https://inoreader.com/) - One of the biggest RSS readers and news aggregators out there. 
+  - [Inoreader.com](https://inoreader.com/) - One of the biggest RSS readers and news aggregators out there.
```
Comment (Scrum) [inline README.md]: @johndatserakis Needs to be at the end of the list.

File: README.md [hunk 3]
```diff
@@ -1411,7 +1412,7 @@ Tooltips / popovers
  - [vue-audio-recorder](https://github.com/grishkovelli/vue-audio-recorder) - Audio recorder for Vue.js. It allows to create, play, download and store records on a server
  - [vue-video-section](https://github.com/johndatserakis/vue-video-section) - A simple video header/section component for Vue. Good for video backgrounds and overlaying content on them.
  - [vue-waveform](https://github.com/chenqiaoen521/vue-waveform) -It is use canvas waveform audio player wavesurfer component  contains timeline bar line for Vue 2.x
- - [vue-responsive-video-background-player](https://github.com/avidofood/vue-responsive-video-background-player) - Play your own videos in background responsively in different resolutions. 
+ - [vue-responsive-video-background-player](https://github.com/avidofood/vue-responsive-video-background-player) - Play your own videos in background responsively in different resolutions.
```
Comment (Scrum) [inline README.md]: @johndatserakis Needs to be at the end of the list.

**Comment by johndatserakis** [comment]
Hey @Scrum, sorry for the confusion.

**Comment by johndatserakis** [comment]
Hey there @Scrum,

Sorry for the confusion. The other lines were not added by me and are not my projects, it seems like they just got mixed in on my recent merge.

So to make sure everything's nice and clean, I removed all my previous changes, merged in a fresh copy of master, and added back my entry so there's a clean diff with only my single line change. It should be ok now.

Sorry and thank you.

**Pull request merged**